### PR TITLE
Add generateSendingAllTx API

### PR DESCRIPTION
**Pull request by classicalliu**
Waiting for https://github.com/nervosnetwork/neuron/pull/1173

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/api.ts [hunk 1]
```diff
@@ -184,6 +184,19 @@ export default class ApiController {
     return WalletsController.generateTx(params)
   }
 
+  @MapApiResponse
+  public static async generateAllTx(params: {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/controllers/api.ts]: The name `generateAllTx` is confusing.

File: packages/neuron-wallet/src/models/transaction-fee.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export default class TransactionFee {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/models/transaction-fee.ts]: It seems merged in the develop branch, please rebase on it first.

Comment (classicalliu) [reply packages/neuron-wallet/src/models/transaction-fee.ts]: It's a draft PR and didn't request review yet .....

File: packages/neuron-wallet/src/controllers/api.ts (hunk 1 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/api.ts]: Any idea ?

**Comment by Keith-CY** [comment]
CI failed

**Comment by classicalliu** [comment]
> CI failed

Passed now

**Pull request merged**